controllers/operatorpolicy_controller.go

```diff
@@ -216,14 +221,16 @@ func (r *OperatorPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		errs = append(errs, err)
 	}
 
+	if !reflect.DeepEqual(policy.Status, originalStatus) {
```

Comment:
Optional optimization:
```suggestion
	if conditionChanged || !reflect.DeepEqual(policy.Status, originalStatus) {
```